### PR TITLE
Update input label to passthrough props

### PR DESCRIPTION
**Pull request by stredger**
## Description

Update InputLabel to take and passthrough some Typography 

## Screenshots
Screen Shot 2021-04-29 at 4 56 37 PM
Screen Shot 2021-04-29 at 4 56 43 PM

### CODE REVIEW

File: packages/riipen-ui/src/components/InputLabel.jsx [hunk 1]
```diff
@@ -3,123 +3,105 @@ import PropTypes from "prop-types";
...
+        label {
+          color: ${theme.palette.text.secondary};
+          display: inline-block;
+          font-family: ${theme.typography.body1.fontFamily};
+          font-size: 16px;
+          font-weight: ${theme.typography.body1.fontWeight};
+          letter-spacing: ${theme.typography.body1.letterSpacing};
+          line-height: 1.4;
+          margin-bottom: ${theme.spacing(marginBottom)}px;
+        }
```
Comment (stredger) [inline packages/riipen-ui/src/components/InputLabel.jsx:48]: We could move this into a typography variant. i left it here so we can leave the default props the way they are

File: packages/riipen-ui-docs/src/pages/components/input/LabelProps.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import Form from "riipen-ui/components/Form";
+import Input from "riipen-ui/components/Input";
+
+export default function Color() {
```
Comment (ksmort) [inline packages/riipen-ui-docs/src/pages/components/input/LabelProps.jsx:6]:
```suggestion
export default function LabelProps() {
```

File: packages/riipen-ui/src/components/InputLabel.jsx [hunk 3]
```diff
@@ -3,123 +3,105 @@ import PropTypes from "prop-types";
...
+          </Typography>
+        </label>
+      )}
+      {hint && <InputHint color={color}>{hint}</InputHint>}
+      <style jsx>{`
+        label {
+          color: ${theme.palette.text.secondary};
+          display: inline-block;
+          font-family: ${theme.typography.body1.fontFamily};
+          font-size: 16px;
```
Comment (ksmort) [inline packages/riipen-ui/src/components/InputLabel.jsx:43]: This is possibly h5 font size?

**Review by ksmort — APPROVED** [review]
Looks 🔥

File: packages/riipen-ui/src/components/InputLabel.jsx (hunk 3 above)
Comment (stredger) [reply packages/riipen-ui/src/components/InputLabel.jsx:43]: oh you are 100% correct. that makes this much easier actually, i can just use h5 as the default variant and probably axe all this css

**Review by jordanell — APPROVED** [review]

**Pull request merged**